packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -72,9 +72,7 @@ export async function activate(extensionContext: vscode.ExtensionContext) {
   await workspaceContext.initialize(extensionContext);
 
   // Telemetry
-  const extensionPackage = require(extensionContext.asAbsolutePath(
-    './package.json'
-  ));
+  const extensionPackage = extensionContext.extension.packageJSON;
```

Comment:
So when bundling it changes the structure of the extension (this `/dist/index.js ` vs `out/src/index.js`). I found this extensionContext value that supplies the package json so I updated to use it.  Required a bump of the vscode types version.